Library/Homebrew/env_config.rb

```diff
@@ -37,6 +37,12 @@ module EnvConfig
                      "to instead be downloaded from " \
                      "`http://localhost:8080/v2/homebrew/core/gettext/manifests/0.21`",
       },
+      HOMEBREW_API_AUTO_UPDATE_SECS:             {
+        description: "Check Homebrew's API for new formulae or cask data every " \
+                     "`HOMEBREW_API_AUTO_UPDATE_SECS` seconds. Alternatively, disable API auto-update " \
+                     "checks entirely with HOMEBREW_NO_AUTO_UPDATE.",
+        default:     1800,
```

Comment:
```suggestion
        default:     3600,
```
Not sure it's useful to update this more often than the API is updated.

Reply:
@carlocab I disagree. If you set it to 3600 and your last update was e.g. 1 second before: you're waiting 3600 more until you see any updates. Setting it to half that gets the balance meaning folks are likely to see an update more quickly if they just missed one without over optimising for too many updates.

Reply:
Is it that important for users to get updates sooner?

Setting it to `1800` means that half of all update attempts are no-ops, and doubles the number of times we hit the API endpoint. The latter feels like something to be particularly wary of given the server-side issues we saw last week (but I admittedly have very little visibility regarding this issue).

Reply:
> Yes. Maintainer time is our most precious resource. That's why we've previously had auto-updates set to 5 minutes.

In that case, shouldn't we regenerate the API data more frequently too?

> It's statically generated and behind a CDN so: nope, this is not relevant. We are doing `git fetch` way more than that and it's way more expensive.

It was also statically generated and behind a CDN last week, and yet `git fetch` turned out to be much more reliable, so it seems to me that computational/network expense might be the wrong cost model here.

Reply:
> In that case, shouldn't we regenerate the API data more frequently too?

Yeh, we could. The job takes about 15m but we could regenerate every 20-25m instead of 30m. 

There's also definitely room to improve the generation performance there.

> It was also statically generated and behind a CDN last week, and yet git fetch turned out to be much more reliable, so it seems to me that computational/network expense might be the wrong cost model here.

Yeh, there were bugs there (not just affecting us) that have since been addressed. Hitting the "API" more often is not going to cause more issues.